odoosentinel/__init__.py

```diff
@@ -126,6 +132,23 @@ def scanner_check(self):
         if isinstance(self.scenario_id, list):
             self.scenario_id, self.scenario_name = self.scenario_id
 
+    def _resize(self, initial_screen_size):
+        """
+        Resizes the window
```

Comment:
Please remove the extra spaces & add a period at the end of the statement. See [PEP-257](https://www.python.org/dev/peps/pep-0257/) for info on docblocks

```
"""Resizes the window."""
```